src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -161,6 +162,7 @@ export const CheckoutPayment = ({
   })
   const { search: is3DsComplete } = useLocation<{ search: string }>()
   const [isLoading, setIsLoading] = useState(false)
+  const [isPageLoading, setIsPageLoading] = useState(false)
```

Comment:
Should we name this differently? It's a little hard to understand the different meanings of "isLoading" and "isPageLoading" -- no?

Reply:
hmm I guess pageLoading is meant for the big spinner where page is unusable while isLoading is meant for quote recreation. we can do isDataLoading/ isQuoteLoading instead of isLoading. wyd @robinandeer  ?

Reply:
yeah I think that's sounds clearer :D